rotkehlchen/globaldb/upgrades/v2_v3.py

```diff
@@ -517,6 +517,7 @@ def migrate_to_v3(connection: 'DBConnection') -> None:
         cursor.executemany(OWNED_ASSETS_INSERT, owned_assets)
         cursor.executemany(PRICES_INSERT, updated_prices)
         cursor.executemany(BINANCE_INSERT, updated_binance_pairs)
+        cursor.execute(CUSTOM_ASSET_TYPE_INSERT, ('[', 27))
```

Comment:
Is there any need for this insertion to happen with foreign keys turned off?

Reply:
Nah, the placement does not really matter